### DISCUSSION TITLE
Notification area icons don't show up when using Windows 10 bar after upgrading to Win11 23H2

### DISCUSSION THREAD

**Discussion by Hezkezl** [valinet/ExplorerPatcher#3639] [Q&A] [answered]
They worked fine on 21H2, which I had been using for a long time until earlier today. Now the only way I can get specific icons to show up in my notification area is to use the Windows 11 taskbar, which I'm not a fan of since the main reason for using EP is to go back to a Windows 10 taskbar/start menu :(

I had this installed while I was installing the 23H2 upgrade, so I'm not sure if that's the reason why it's not working anymore. But if I go into the taskbar settings and check the 'Other system tray icons' section, the on/off sliders in there don't do anything at all for the Windows 10 bar; they only appear to have an effect on the Windows 11 taskbar. To the point where if I'm using a Win10 bar, toggle some things on, then go back to the Win11 bar, then the icons I toggled will be showing up there. On win10's bar, everything is hidden inside of the ^ carat menu.

Completely clueless about what to do to try to get this to work again, so any help would be appreciated :(

Windows 11 23H2 (OS Build 22631.4037)
Explorer Patcher version 22621.3880.66.5

here's some screenshots of all of my settings.
![image](https://github.com/user-attachments/assets/fe11ac0f-d10b-4332-938d-286c5e1dd9b1)
![image](https://github.com/user-attachments/assets/61953c02-8bb5-4ac5-9017-fcddeb66e736)
![image](https://github.com/user-attachments/assets/fa7750b8-c536-47b7-8647-e6a4ea101602)
![image](https://github.com/user-attachments/assets/e4510e63-3d39-4cb6-9244-5756aeaa682d)
![image](https://github.com/user-attachments/assets/13df8a8c-8885-4799-a6c5-a14509f7ddce)
![image](https://github.com/user-attachments/assets/2e22b64b-9a7d-4424-bf6c-bf3040d5b221)

![image](https://github.com/user-attachments/assets/e80726ba-e655-4d81-ba21-984b82e0b624)
![image](https://github.com/user-attachments/assets/fbb64aac-2850-4d70-b056-85fd6155d9a6)
![image](https://github.com/user-attachments/assets/24d82578-d881-47ba-abc1-1320098c854a)

And here's some examples of the issue I'm having with the taskbar. Here's my windows settings: 
![image](https://github.com/user-attachments/assets/1a70819d-0ebf-461a-8b4f-733669c0fbf6)

And here's the task bar that I see (after clicking the ^ carat menu open) : 
![image](https://github.com/user-attachments/assets/f9a75635-c348-479d-9681-a107374e80fd)

Here's what I see when I switch back to the Windows 11 bar (first settings screenshot, changing Taskbar style from Windows 10 to Windows 11 and then restarting explorer)
![image](https://github.com/user-attachments/assets/63367079-657c-49b8-9b14-03a8f43b2cd7)

I've searched the best I could for other people having similar issues, but either noone else has or I suck at using the search here on GH.

I'm tempted to go back to 21H2 and force Windows 11 to stay there with a group policy edit again, just so that I can have the taskbar being the way that I like it, but for security and stuff it'd obviously be better if I could somehow get it to work on this version :(

**Comment by pyrates999** [comment]
You can drag and drop the icon in the system tray to the popup menu that you don't want shown.

duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1454

Please close this.

**Comment by pyrates999** [comment, marked as answer]
You also enabled the option register as shell extension, disable it by opening up the EP properties application, go to **File Explorer**, and untick **Register as shell extension**.

Here's why: https://github.com/valinet/ExplorerPatcher/wiki/Using-ExplorerPatcher-as-shell-extension

**Comment by Hezkezl** [comment]
There's technically two answers, as both of those look like they work. I unregistered it as a shell extension (which I swore I enabled as part of the testing for why they weren't showing up, though it 100% could've been as to the other issue I commented on and about why the notification window kept crashing all of explorer), and then I was able to drag and drop icons around. I didn't try drag and dropping earlier as I wasn't aware that was something we could do in the first place.

But after doing that, then things sortof started working. The exact picture in my screenshot up in the first post (about the built in Windows 11 show/hide additional tray icons menu) still doesn't work for the Windows 10 (ExplorerPatcher) bar, but the ooold one about Notification Area Icons (screenshot included) DOES work now for showing/hiding things.
![image](https://github.com/user-attachments/assets/83a6cd37-234f-45f8-825f-beacbdf7e05d)


So there's a few workarounds to get things going how I need. Thank you very much, and digging through #1454 I had no idea there were so many other posts about a similar issue, I guess I really do just fail at using Github search... Thanks very much!

**Reply by pyrates999** [reply]
the windows 10 taskbar, both options, can only use the old one in control panel.  It doesn't use the one in settings.
